<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManageBooks</class>
 <widget class="QDialog" name="ManageBooks">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>456</width>
    <height>66</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Manage Books</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="AddBook">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Book</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DeleteBook">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete Book</string>
   </property>
  </widget>
  <widget class="QPushButton" name="FindBook">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Find Book</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Issue Book</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
